ez-toolbar.html

```diff
@@ -9,6 +9,10 @@
 
```

Comment:
please move to comment above to the JS file and it's a good opportunity to complete with a small description of `ez-toolbar` and how it behaves